docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
+We've introduced 2 new columns in the [project list view](https://pipedream.com/projects): **Owner** and **Access**.
+
+**Owner** indicates who within the workspace owns each project. This is typically the person who created the project.
+
+![Project Listing (Owner)](./images/project-listing-owner.png)
+
+::: warning Some projects may not have owners
+Projects created before February 2024 don't automatically have owners.
```

Comment:
Consider adding a short sentence of a suggestion to take if your project doesn't have an owner.

If there's no recommended action, then reassure the user that this is still fine, but these are the consequences of that.